### PR TITLE
Updated ‘becomes maneuver’ for guidance g11n

### LINKED ISSUE #274: Add 'becomes' test and remove old form street names method

### PR DESCRIPTION
**Pull request by dgearhart**
Closes #274

### CODE REVIEW

File: locales/de-DE.json [hunk 1]
```diff
@@ -78,18 +78,16 @@
...
       },
-      "empty_street_name_labels": ["Fußweg", "Radweg", "Mountainbike Strecke"],
       "example_phrases": {
         "0": ["Vine Street becomes Middletown Road."]
       }
     },
     "becomes_verbal": {
       "phrases": {
-        "0": "<PREV_STREET_NAMES> wird zu<STREET_NAMES>."
+        "0": "<PREVIOUS_STREET_NAMES> wird zu<STREET_NAMES>."
```
Comment (gknisely) [inline locales/de-DE.json]: space here after zu?

Comment (dgearhart) [inline locales/de-DE.json]: fixed - thanks

File: locales/de-DE.json [hunk 2]
```diff
@@ -78,18 +78,16 @@
...
       },
-      "empty_street_name_labels": ["Fußweg", "Radweg", "Mountainbike Strecke"],
       "example_phrases": {
         "0": ["Vine Street becomes Middletown Road."]
       }
     },
     "becomes_verbal": {
       "phrases": {
-        "0": "<PREV_STREET_NAMES> wird zu<STREET_NAMES>."
+        "0": "<PREVIOUS_STREET_NAMES> wird zu <STREET_NAMES>."
```
Comment (gknisely) [inline locales/de-DE.json]: space here after zu?

**Comment by gknisely** [comment]
:gb:

**Comment by gknisely** [comment]
:space_invader:

**Pull request merged**